deploy/charts/rig/templates/_config.tpl

```diff
@@ -29,14 +29,20 @@ client:
   {{- end }}
 cluster:
   type: k8s
+  {{- with .Values.rig.cluster.dev_registry }}
+  dev_registry:
```

Comment:
is `default_registry` better? I know this was my idea, but I'm not sure what's actually "dev" about it :D

Reply:
I guess the 'dev' thing is because it is only meant for development, not for any production purporses. I'm not really building it with scalability or security in mind (becomes relevant with PVs)